snc-library.sh

```diff
@@ -184,22 +184,18 @@ function create_pvs() {
 
     # Add registry storage to pvc
     retry ${OC} patch config.imageregistry.operator.openshift.io/cluster --patch='[{"op": "add", "path": "/spec/storage/pvc", "value": {"claim": "crc-image-registry-storage"}}]' --type=json
-    # Remove emptyDir as storage for registry
-    retry ${OC} patch config.imageregistry.operator.openshift.io/cluster --patch='[{"op": "remove", "path": "/spec/storage/emptyDir"}]' --type=json
```

Comment:
This is removed because as part of SNO config image registry doesn't have `/spec/storage/emptyDir` object.